x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/document_fields_header.tsx

```diff
@@ -15,7 +15,7 @@ export const DocumentFieldsHeader = () => {
       <EuiTitle size="s">
         <h2>
           {i18n.translate('xpack.idxMgmt.mappingsEditor.documentFieldsTitle', {
-            defaultMessage: 'Document fields',
+            defaultMessage: 'Mapped fields',
```

Comment:
I noticed it's a little repetitive now with the tab and title. Thoughts?

<img width="194" alt="Screen Shot 2019-12-16 at 12 01 17 PM" src="https://user-images.githubusercontent.com/5226211/70927401-e1477e00-1ffc-11ea-9ba5-dae2695d24ef.png">